src/graphql/resolvers/userResolver.js

```diff
@@ -15,9 +16,37 @@ const resolver = {
 
     User: {
         async stockTransactions(parent, args) {
-            const stockTransactions = await StockTransaction.find({user: parent._id})
+            const stockTransactions = await StockTransaction.find({ user: parent._id })
             return stockTransactions
         }
+    },
+
+    Mutation: {
+        async register(_, { name, password, confirmPassword, email, bits_id }, context, info) {
+            const user = new User({
+                name,
+                email,
+                password,
+                bits_id
+            })
+            await user.generateAuthToken()
+            await user.save()
+            return user
```

Comment:
Returning just the user won't do. The client needs to get the token also. So add the token you generated to the response.

Reply:
Return the token separately. Not inside the user object. Fix this.